ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/BackupReplayFile.java

```diff
@@ -84,6 +84,15 @@ public void write(final ConsumerRecord<byte[], byte[]> record) throws IOExceptio
       throw new IOException("Write permission denied.");
     }
 
+    if (record == null) {
+      throw new IllegalArgumentException("Record is null");
```

Comment:
when can this ever happen? can we throw a more useful error message?

Reply:
Because any records read from the cmd topic should not have null records, I feel this can only happen if the local file was manipulated somehow?

Reply:
I don't think this never happen. I added just in case, but seems unnecessary. I removed it.